manifests/bucketeer/charts/batch/templates/cronjob.yaml

```diff
@@ -0,0 +1,199 @@
+    heritage: {{ .Release.Service }}
+spec:
+  schedule: {{ .Values.cronjob.experimentStatusUpdaterSchedule }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: experiment-status-updater
+              image: curlimages/curl:8.1.2
+              imagePullPolicy: IfNotPresent
+              command:
+                - "curl"
+              args:
+                - "-X"
+                - "POST"
+                - "-d"
+                - '{"job": "ExperimentStatusUpdater"}'
+                - "-H"
+                - "authorization: bearer {{ template "service-token-secret" . }}"
+                - "https://web-gateway.{{ .Values.namespace }}.svc.cluster.local:{{ .Values.cronjob.webGatewayPort }}/bucketeer.batch.BatchService/ExecuteBatchJob"
+          restartPolicy: Never
+
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "batch-server.fullname" . }}-experiment-running-watcher
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ template "batch-server.name" . }}
+    chart: {{ template "batch-server.chart" . }}
+    release: {{ template "batch-server.fullname" . }}
+    heritage: {{ .Release.Service }}
+spec:
+  schedule: {{ .Values.cronjob.experimentRunningWatcherSchedule }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: experiment-running-watcher
+              image: curlimages/curl:8.1.2
+              imagePullPolicy: IfNotPresent
+              command:
+                - "curl"
+              args:
+                - "-X"
+                - "POST"
+                - "-d"
+                - '{"job": "ExperimentRunningWatcher"}'
+                - "-H"
+                - "authorization: bearer {{ template "service-token-secret" . }}"
+                - "https://web-gateway.{{ .Values.namespace }}.svc.cluster.local:{{ .Values.cronjob.webGatewayPort }}/bucketeer.batch.BatchService/ExecuteBatchJob"
+          restartPolicy: Never
+
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "batch-server.fullname" . }}-feature-state-watcher
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ template "batch-server.name" . }}
+    chart: {{ template "batch-server.chart" . }}
+    release: {{ template "batch-server.fullname" . }}
+    heritage: {{ .Release.Service }}
+spec:
+  schedule: {{ .Values.cronjob.featureStateWatcherSchedule }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: feature-state-watcher
+              image: curlimages/curl:8.1.2
+              imagePullPolicy: IfNotPresent
+              command:
+                - "curl"
+              args:
+                - "-X"
+                - "POST"
+                - "-d"
+                - '{"job": "FeatureStateWatcher"}'
+                - "-H"
+                - "authorization: bearer {{ template "service-token-secret" . }}"
```

Comment:
Because the service token is mounted in the container, you must access it from the file configured here.

https://github.com/bucketeer-io/bucketeer/pull/455/files#diff-299980ca0f3ebbce9876b6dabecf46bed633a07f1206ad49a93eb68379924c7cR98

Then, you should use it like below.

https://github.com/bucketeer-io/bucketeer/pull/453/files#diff-2a427e5e8f0b1a14e40f56488e9eb80dafdb58fec3335429b6018a473cc2b08fR29

Same for the other jobs.

Reply:
Done!